lib/screens/splash_screen.dart

```diff
@@ -41,11 +40,11 @@ class _SplashScreenState extends State<SplashScreen>
     _animation.addListener(() => this.setState(() {}));
     _animationController.forward();
 
-    loadAssets()
-        .then((value) => Navigator.pushReplacementNamed(context, '/home'));
-    // Timer(Duration(seconds: 3), () {
-    //   Navigator.pushNamed(context, '/home');
-    // });
+    loadAssets().then((value) {
+      Timer(Duration(seconds: 2), () {
+        Navigator.pushReplacementNamed(context, '/home');
+      });
+    });
```

Comment:
Why is there a timer here?
I'm guessing its for a smoother animation if loadAssets() is too fast?

Reply:
Yes exactly this.